CryptoTechReminderSystem.Test/UseCase/ShameLateDevelopersTests.cs

```diff
@@ -54,26 +54,25 @@ public void CanRemindDevelopers()
         [Test]
         [TestCase( "W0123CHAN", "W123AMON", "W789ROSS")]
         [TestCase( "W0123CHAN", "W123AMON")]
-        public void CanCheckMessageHasAllUsers(params string[] userId)
+        public void CanCheckShameMessageHasAllUsers(params string[] userId)
         {
-            
             var getLateDevelopersStub = new GetLateDevelopersStub(userId.ToList());
             var shameLateDevelopers = new ShameLateDevelopers(getLateDevelopersStub, _sendReminderSpy);
-            var expectedMessage = "TIMESHEETS ARE GOOD YO!";
+            var expectedShameMessage = "TIMESHEETS ARE GOOD YO!";
 
             shameLateDevelopers.Execute(
                 new ShameLateDevelopersRequest
                 {
-                    Message = expectedMessage
+                    ShameMessage = expectedShameMessage
                 }
             );
 
             foreach (var user in userId)
             {
-                expectedMessage += $"\n• <@{user}>";
+                expectedShameMessage += $"\n• <@{user}>";
             }
```

Comment:
Ran `option + enter` using JetBrains Rider for this `foreach` statement and got this back. Looks nice, to me anyway.

```csharp
expectedShameMessage = userId.Aggregate(expectedShameMessage, (current, user) => current + $"\n• <@{user}>");
```